api/DTOs/TimeEntryDTO.cs

```diff
@@ -18,6 +18,7 @@ public TimeEntryDTO(TimeEntry timeEntry, Leave? leave)
             WorkedHours = timeEntry.WorkedHours?.Remove(timeEntry.WorkedHours.LastIndexOf(':'));
             TrackedHours = timeEntry.TrackedHours.ToString(@"hh\:mm");
             User = timeEntry.User;
+            Avatar = $"{domain}/media/{timeEntry.User.ProfileImage!.CollectionName}/{timeEntry.User.ProfileImage!.FileName}";
```

Comment:
```suggestion
           User = new UserDTO(timeEntry.User, domain);
```
If `User` and `domain` is available in DTO, you can instead use `UserDTO` so you won't need to create new `Avatar` attribute